support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.jsx

```diff
@@ -313,7 +311,7 @@ function PaperCheckoutForm(props: PropTypes) {
               setTelephone={props.setTelephone}
               formErrors={props.formErrors}
               signOut={props.signOut}
-              isUsingGuestCheckout={isUsingGuestCheckout}
+              isUsingGuestCheckout
```

Comment:
We still need to pass this prop for the time being because this component is shared between checkouts